src/react-apps/applications/runtime/src/containers/Render.tsx

```diff
@@ -14,7 +14,7 @@ export class RenderComponent extends React.Component<IRenderProps, null> {
     return (
       <div className='col-12'>
         {layout && layout.map((component: any) => {
-          if (component.type === 'Container') {
+          if (component.component === 'Container') {
```

Comment:
There is no type, only component and itemType. (as it is in the old version)

Reply:
Maybe we should map it when we convert. Component.component does not make much sense to me.

Reply:
https://github.com/Altinn/altinn-studio/pull/1601 is already approved...

Reply:
This wasn't straight forward as the converting of the layout is done when the page is loading and it's not updating when new components are added. Made a new issue #1776 because I don't want to edit too much in the ux-editor on this branch.